plugins/Live/API.php

```diff
@@ -194,9 +211,8 @@ public function getVisitorProfile($idSite, $visitorId = false, $segment = false)
         $newSegment = ($segment === false ? '' : $segment . ';') . 'visitorId==' . $visitorId;
 
         $visits = $this->loadLastVisitorDetailsFromDatabase($idSite, $period = false, $date = false, $newSegment,
-            $numVisitorsToFetch = self::VISITOR_PROFILE_MAX_VISITS_TO_AGGREGATE,
-            $overrideVisitorId = false,
-            $minTimestamp = false);
+            $offset = 0,
+            $limit = self::VISITOR_PROFILE_MAX_VISITS_TO_AGGREGATE);
         $this->addFilterToCleanVisitors($visits, $idSite, $flat = false, $doNotFetchActions = false, $filterNow = true);
```

Comment:
Removed the last two parameters from the function call as they were optional anyway and there names were actually wrong